starcompactor/machine_event_dump.py

```diff
@@ -111,72 +115,85 @@ def main(argv):
         'port': 3306,
     })
     mysqlargs.inject(parser)
-    parser.add_argument('-c', '--epoch', type=str, default=Constants.CHAMELEON_KVM_START_DATE + 'T00:00:00',
-        help='Time to compute relative dates from')
-    parser.add_argument('-m', '--hashed-masking-method', type=str, default='sha2-salted',
-        choices=trans.MASKERS,
-        help='Hashed mask method (for host name). "sha1-raw" is legacy and not recommended as '
-             'it is vulnerable to cracking. "none" is for debugging only.')
-    parser.add_argument('-s', '--hashed-masking-salt', type=str,
-        help = 'Salt of hashed masking method (for host name). Please use the same salt for instance event host name! '
-               'Ignored if host name mask type is "none".')
-    parser.add_argument('-j', '--jsons', action='store_true',
+    parser.add_argument('--hashed-masking-method', type=str, default='sha2-salted', choices=trans.MASKERS,
+        help='Hashed mask method (for host name). "sha1-raw" is legacy and not recommended as it is vulnerable to cracking. "none" is for debugging only.')
+    parser.add_argument('--hashed-masking-salt', type=str, default=None,
+        help = 'Salt of hashed masking method (for host name). Please use the same salt for instance event host name! Ignored if host name mask type is "none".')
+    parser.add_argument('--instance-type', type=str, default='vm', choices=['vm', 'baremetal'],
+        help='Type of the instance. Choose vm or baremetal')
+    parser.add_argument('--jsons', action='store_true',
         help='Format output as one JSON per line (defaults to CSV-style)')
-    parser.add_argument('-v', '--verbose',
-        action='store_const', const=logging.INFO, dest="loglevel",
+    parser.add_argument('--verbose', action='store_const', const=logging.INFO, dest="loglevel",
         help='Increase verbosity about the dump.')
-    parser.add_argument('--debug',
-        action='store_const', const=logging.DEBUG, dest="loglevel",
+    parser.add_argument('--debug', action='store_const', const=logging.DEBUG, dest="loglevel",
         help='Debug-level logging.')
     parser.add_argument('output_file', type=str,
         help='File to dump results')
 
     args = parser.parse_args(argv[1:])
     mysqlargs.extract(args)
+    
+    LOG.debug('instance type: {}'.format(args.instance_type))
```

Comment:
This line will never be printed because the `--debug` flag is evaluated later. This should move to after the `logging.basicConfig` section is set up.